docs/contributing/roadmap.rst

```diff
@@ -0,0 +1,107 @@
+.. _roadmap:
+
+####################
+Development Roadmap
+####################
+
+**Authors**: Anemoi Technical Subgroup (ATS)
+
+**Date**: May 2025
+
+This roadmap serves as a guideline for the future development of the Anemoi project. 
+We aim to maintain flexibility and adaptability, ensuring that the system can evolve
+to meet emerging needs while maintaining a high level of performance and usability.
+The collaboration of diverse institutions will continue to be a core strength, ensuring
+the project meets the evolving demands of weather and climate modeling.
+
+**************
+Our Philosophy
+**************
+
+We aim to build a flexible, extensible, and modular ecosystem that addresses the needs
+of weather and climate modeling through machine learning. By focusing on
+generalization, interoperability, and ease of integration, our project aims to
+support diverse use cases, from regional forecasting to global climate modeling.
+This ecosystem thrives on contributions from diverse institutions and aims to 
+remain domain-agnostic while meeting the specific needs of different stakeholders.
+
+**********************
+Key Development Themes
+**********************
+
+The technical direction for this project focuses on extending its usability, modularity,
+and flexibility across different domains and datasets. We have identified 4 key areas
+for improvement and expansion.
+
+1. **Flexible Model Architectures**
+
+   - **Current Focus**:
+
+     - Enhance model architectures to be more modular.
+     - Focus on implementing probabilistic and ensemble forecasting techniques (e.g.,
+       CRPS Loss, diffusion).
+
+   - **Long-Term Vision**:
+
+     - Develop a robust framework for model sharing across various ecosystems, 
+       supporting different types of models.
+     - Ensure that models can easily scale from local to global forecasting tasks while
+       maintaining computational efficiency.
+
+2. **Data Handling**
+
+   - **Current Focus**:
+
+     - Support for multiple datasets, particularly for high-resolution observational
+       data (e.g., radar, satellite, SYNOP).
+     - Implement multiple encoders and decoders.
+
+   - **Long-Term Vision**:
+
```

Comment:
In the first part, it´s mentioned that Anemoi will also support climate modelling, while so far I don't think we have build/registered any climate datasets. Could it be worth adding that for the long-term vision?

Reply:
that's maybe a good thing for discussion in the ATS, do we want to explicitly mention the climate modelling use case?

Reply:
was this discussed already, otherwise we could bring it up this week

Reply:
Not yet discussed.

Reply:
OK let me remove the climate use case for now. We can always include it later on if we decide to make it a focus.